### ISSUE TITLE
Breaks on Sphinx 1.7.0

### ISSUE THREAD

**Issue by SunDwarf** [python-trio/sphinxcontrib-trio#7]
```
(curious-yiBjxk2Q) $ make html                                                                                                                                                             
Running Sphinx v1.7.0b2

Extension error:
Could not import extension sphinxcontrib_trio (exception: cannot import name 'Options' from 'sphinx.ext.autodoc' (/home/laura/.local/share/virtualenvs/curious-yiBjxk2Q/lib/python3.7/site-packages/sphinx/ext/autodoc/__init__.py))
make: *** [Makefile:20: html] Error 2
```

Options seems to have been removed from sphinx.ext.autodoc (but it's not in the changelog?)

**Comment by njsmith** [comment]
Filed upstream, since 1.7.0 is still a pre-release and they still have a chance to fix it before the real release: https://github.com/sphinx-doc/sphinx/issues/4538

It wouldn't be hard to work around in sphinxcontrib-trio too – all we use the `Options` class for is the incantation `self.options = Options(self.options)`. We could make it `self.options = self.options.__class__(self.options)` and skip the import entirely. Would you rather do that, or downgrade to a non-beta version of sphinx while waiting to see if they fix the regression?

**Comment by SunDwarf** [comment]
I was mostly using 1.7 because (a few versions ago) the latest version of 1.6 wasn't parsing doc comments in files with f-strings.

Since 1.6.6 seems to fix that, I can downgrade to 1.6.6 for now.

**Comment by njsmith** [comment]
You have a workaround and sphinx has fixed this upstream, so I guess this can be closed. Thanks for reporting! If you hadn't caught this it would have broken for a lot of people when 1.7.0 came out.
